xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.h

```diff
@@ -222,6 +222,7 @@ class CRenderManager
 
   struct SPresent
   {
+    SPresent();
```

Comment:
maybe initialise with something like PRESENT_METHOD_INVALID. that makes it clearer that the linked render buffer does not hold any valid frame.